css-fonts-5/Overview.bs

```diff
@@ -168,19 +168,42 @@ For: @font-face
 Initial: normal
 </pre>
 
-The 'ascent-override', 'descent-override' and 'line-gap-override' descriptors define the
+<pre class='descdef'>
+Name: advance-override
+Value: normal | <<percentage>>
+For: @font-face
+Initial: normal
+</pre>
+
+The 'ascent-override', 'descent-override', and 'line-gap-override' descriptors define the
 <a spec="CSS-INLINE-3">ascent metric</a>, <a spec="CSS-INLINE-3">descent metric</a> and
-<a spec="CSS-INLINE-3">line gap metric</a> of the font, respectively.
+<a spec="CSS-INLINE-3">line gap metric</a> of the font, respectively. The `advance-override`
+descriptor adjusts the <dfn export>tracking</dfn> between adjacent
+<a spec="CSS-TEXT-3>typographic character units</a>.
 
 When the descriptor value is 'normal', the corresponding metric value is obtained from the
 font file directly.
 
 Note: User agents may draw data from different places from the font file as the metric values,
 which results in different text layouts.
 
-When the descriptor value is a percentage, the corresponding metric value is resolved as the
+For the 'ascent-override', 'descent-override' and 'line-gap-override' descriptors, when the
+descriptor value is a percentage, the corresponding metric value is resolved as the
 given percentage multiplied by the used font size. Negative values are invalid at parse time.
 
+For the 'advance-override' descriptor, when the descriptor value is a percentage, the
+spacing between adjacent <a>typographic character units</a> is multiplied by the percentage.
```

Comment:
Is it the spacing between, that is multiplied? It seems that it is the total advance width, (which include the part taken up by the glyph) which is multiplied. If the percentage is greater than 100, that will have the effect of increasing the spacing between TCUs.

That probably needs a diagram, (later, not to land this pull request)

Reply:
Fixed in https://github.com/w3c/csswg-drafts/pull/5991/commits/8cd3dd90e1e582dc0fd54c7799845a2588f37e75.

An example with diagrams is in https://github.com/w3c/csswg-drafts/pull/5991/commits/a6e10d5130ff746374ee54fc28857b541037d7e1.